htmresearch/frameworks/classification/classification_network.py

```diff
@@ -459,6 +487,33 @@ def _getClassifierInference(classifierRegion):
 
 
 
+def _inspectTMPredictionQuality(tm, numRecordsToInspect):
+  """Inspect prediction quality of TM over the most recent N records"""
```

Comment:
@ywcui1990 By most recent N records, you mean "most recent numRecordsToInspect", right?